usermanagerlib/src/main/java/org/pesmypetcare/usermanagerlib/clients/TaskManager.java

```diff
@@ -142,26 +143,31 @@ private void writeRequestBody(HttpURLConnection con) throws IOException {
         writer.close();
     }
 
-    /*
-     * Method that fills the Body of a response
-     * @return void
+    /**
+     * The method to get the response of the request.
+     * @param con The HttpURLConnection where the request comes from.
+     * @return Return the body of the response.
+     * @throws IOException When Input or Outpul fails
      */
-    private StringBuilder getResponseBody(HttpURLConnection con) throws IOException {
-        BufferedReader in = new BufferedReader(new InputStreamReader(
-            con.getInputStream()));
-        String inputLine;
-        StringBuilder response = new StringBuilder();
-
-        while ((inputLine = in.readLine()) != null) {
-            response.append(inputLine);
-        }
-        in.close();
-        return response;
-    }
-
-    /*
-     * Method that stablishes an HTTP connection with the provided URL with the method
-     * @return void
+     private StringBuilder getResponseBody(HttpURLConnection con) throws IOException {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['method def modifier' has incorrect indentation level 5, expected level should be 4.](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES_My-Pet-Care-API/pullRequest?prid=5257641)